sympy/parsing/autolev/__init__.py

```diff
@@ -0,0 +1,94 @@
+from sympy.external import import_module
+
+
+def parse_autolev(inp, output=None):
```

Comment:
What version(s) of autolev code does this parse?

Reply:
I have followed the Autolev Tutorial and the Autolev version you sent me (these are version 4.1) and Dynamics Online (which uses an older version) as guides. I didn't find too much of a difference when I used the older codes in the newer version but I think I went with the newer version in case of a conflict or deprecation warning. So I would say 4.1.

Reply:
Ok, stating clearly that we are following the autolev 4.1 spec is good.